src/lib/accessibility/Behaviors/Chat/chatBehavior.ts

```diff
@@ -34,4 +34,13 @@ const ChatBehavior: Accessibility = (props: any) => ({
   },
 })
 
+const getLastTabbableElement = (root: HTMLElement): HTMLElement => {
+  const chatItemsElements = root.querySelectorAll(
+    `[${CHAT_FOCUSZONE_ATTRIBUTE}] .ui-chat__item > [${IS_FOCUSABLE_ATTRIBUTE}]`,
+  )
+  return (
```

Comment:
note this will return `false` in case if condition is not satisfied. For the sake of preserving type of return value, it is better to explicitly return `null` instead:

```js
(chatItemsElements.length > 0 
  ? chatItemsElements[chatItemsElements.length - 1]
  : null)  as HTMLElement
```

Reply:
agree, thank you